CHANGELOG.md: Add 0.6.0 entries

dmajda opened the pull request.

dmajda commented:
@demmer

demmer commented on CHANGELOG.md:
```diff
@@ -2,20 +2,56 @@
+
+### Minor Changes
+
+- The `.` operator can now be used to access object properties (not just module exports). [[#497](https://github.com/juttle/juttle/pull/497)]
```
I think it'd be useful to clarify what situations this extends the language to support nested points support for, i.e. rvalues for `put`, assignments in functions, etc.

Depending on how useful this is, I'd consider this a major change perhaps.

demmer commented:
looks good otherwise.

dmajda commented:
@demmer I clarified the `.` operator entry and moved it to Major Changes (I was split on this myself). I also extended the `sigma` changes to cover stdlib and added entries for work done since yesterday.

@bkutil Could you check the above so that I can release? (It’s the last 3 commits.)

dmajda commented:
(I’ll squash before merge.)

bkutil commented:
The extra changes look ok. +1
